_posts/2020-03-04-cockpit-214.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 214
+author: pitti
+date: 2020-03-04
+tags: cockpit linux
+slug: cockpit-214
+category: release
+summary: Cockpit with bug fixes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 214.
+
+### Networking: List Firewall active zones when unprivileged
+
+[firewalld](https://firewalld.org/) recently became more restrictive. Listing services in a zone is no longer possible for anyone but superusers. As a result, the Firewall page was empty in unprivileged mode.
+
+With this new Cockpit version, the Firewall page shows active zones again, however without services listed.
+
+### Selenium tests deprecation
+
+The Cockpit team has adjusted many "verify" browser tests to be non-destructive. This reworking was done to allow test VMs to be reused without resetting, which was our primary remaining reason for still having the Selenium tests.
+
+As the Selenium are much harder to debug than verify tests and coverage is not quite as complete, we will not develop them further. Eventually, the Selenium tests will be completely removed.
```

Comment:
```suggestion
Compared to verify tests, Selenium tests are much harder to debug and coverage is not quite as complete. Therefore, we will no longer develop Selenium tests. Eventually, the Selenium tests will be completely removed.
```

Missing the word "tests" after Selenium and the flow was a bit clunky still. Revised to include both. Additionally, I specified Selenium once more as it was not clear which tests will be abandoned. Clarity is more important than redundancy. :wink: